assets/javascript/app.js

```diff
@@ -0,0 +1,77 @@
+
+$(document).ready(function() {
+
+  var number = 60;
+  
+  var intervalId;
+
+  var running = false;
+  
+  
+  function run() {
+    if (!running) {
+      running = true;
+      intervalId = setInterval(decrement, 1000);
+    }
+  }
+
+  function decrement() {
+    number--;
+    $("#show-number").html("<h2>" + number + "</h2>");
+    if (number === 0) {
+      
+      stop();
+      var userAnswers = [];
+    // Loops through all checked input fields
+    $('input[name="pokemon"]:checked').each(function() {
+      // DONE - 1. Get the 'value' of the checked radio button
+      // DONE - 2. Push that 'value' to the userAnswers array
+      userAnswers.push($(this).val())
+      
+    })
+    // DONE - 3. Then checkAnswer(userAnswers) function
+    checkAnswer(userAnswers)
+      alert("Time Up!"); 
+    }
+  }
+
+  function stop() {
+    running = false;
+    clearInterval(intervalId);
+    
+  }
+ 
+
+  function checkAnswer(userAnswers) {
+    var correctAnswers = ["true", "true", "true", "true"];
+
+    for (var i = 0; i < correctAnswers.length; i++) {
```

Comment:
I suggest adding some sort of *check if answered* verification. I was able to stop the timer without having to choose any answers.